free-programming-books.md

```diff
@@ -360,8 +360,10 @@
 
 #### Computer Science
 
+* [A New Kind of Science](http://www.wolframscience.com/nks/) - Stephen Wolfram
 * [Computer Science I - Draft](https://bitbucket.org/chrisbourke/computersciencei/src) - Dr. Chris Bourke (PDF) (:construction: *in process*)
 * [Computer Science II - Draft](http://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Dr. Chris Bourke (PDF) (:construction: *in process*)
+* [Foundations of Computer Science](http://www.cl.cam.ac.uk/teaching/2000/FoundsCS/Founds-FP.pdf) - Lawrence C Paulson (PDF)
```

Comment:
There's another book with the same title under Algorithms & Data Structures

Reply:
Also, this book describes itself as not-a-book lecture notes. If we think it still stands alone, we should use the full titel to distinguish it from the author's book.

Reply:
I think we need to discuss more for it in Issue.